File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/uploads/data/CourseUploadList.java
package org.wikiedufoundation.wikiedudashboard.course_detail.uploads.data;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.ToString;

@Getter
@AllArgsConstructor
@ToString
public class CourseUploadList {
    List<CourseUpload> uploads;
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/coursedetail/uploads/data/CourseUploadList.kt
package org.wikiedufoundation.wikiedudashboard.ui.coursedetail.uploads.data

import java.io.Serializable

/**
 * [CourseUploadList] model class
 * @constructor primary constructor
 *
 * @property uploads list of course uploads
 * ***/
data class CourseUploadList(
    val uploads: List<CourseUpload>
) : Serializable
